mender-convert-package

```diff
@@ -150,8 +149,9 @@ temp_img_name=$(basename "$disk_image")
 
 # Strip the image suffix from temp_img_name and set the original image name as
 # the output image name
-image_name="${temp_img_name%.*}-${device_type}-mender"
 
+# replaces whitespace with - to seperate device types. Makes it possible to create files with device_type in the filename   
+image_name="${temp_img_name%.*}-$(sed 's| |-|g' <<< $device_type)-mender"
```

Comment:
Hmm, I'm still not sure about this :thinking:

Reply:
I think `-` is relatively common char in a device type, so consider using `+` instead or something else.

Reply:
Also, do we want the image named image-devicetype1-devicetype2-.....img ? :thinking:  @lluiscampos

Reply:
@oleorhagen  Well, we do have already the device type in the filename (due to the raspberry3/4 incompatible images), so I think we cannot remove it and, therefore, introducing support for multiple devicetypes should be reflected in the filename too. IOW, I agree it is ugly, but I think we must have it.

Then, we could think (and should be fairly easy to implement) to let the user configure the filename with a conf variable, default being as we do today. @NilsOlavKJohansen, are you up to the challenge? (This is classic scope creep on PRs...)

Reply:
Good idea, I am up to the challenge!

Reply:
As discussed in today's meeting, work on this in a separate Pull Request, so that we can merge this one sooner :)